io.github.bothlab.pomidaq.yaml

```diff
@@ -0,0 +1,68 @@
+id: io.github.bothlab.pomidaq
+runtime: org.kde.Platform
+runtime-version: 5.15-21.08
+sdk: org.kde.Sdk
+command: pomidaq
+
+finish-args:
+  - '--share=ipc'
+  - '--socket=wayland'
+  - '--socket=fallback-x11'
+  - '--device=all'
+  - '--filesystem=host'
```

Comment:
this is a very broad filesystem permision

Reply:
Yes, but a lot of scientists have files in a lot of places, for example in our lab data is stored on devices mounted in `/mnt` and directories in `/srv`, other labs have network drives (terrible idea for live recording storage, but still people do that), I have seen people mounting stuff in `/var` etc.
I don't want to limit where data can be exported to, as that will most certainly lead to user frustration.
Maybe marking things like `/usr` as `ro` would be okay as compromise? Otherwise I'd need to add quite a large whitelist of possible locations where scientists could store data...